spec/features/dashboard/collection_spec.rb

```diff
@@ -301,6 +301,51 @@
     end
   end
 
+  describe 'remove works from collection' do
+    context 'user that can edit' do
+      let!(:work2) { create(:work, title: ["King Louie"], member_of_collections: [collection1], user: user) }
+      let!(:work1) { create(:work, title: ["King Kong"], member_of_collections: [collection1], user: user) }
+
+      before do
+        sign_in admin_user
+      end
+      it 'shows remove action buttons' do
+        visit "/dashboard/collections/#{collection1.id}"
+        expect(page).to have_selector('input.collection-remove', count: 2)
+      end
+      it 'removes the first work from the list of items' do
+        visit "/dashboard/collections/#{collection1.id}"
+        expect(page).to have_selector('input.collection-remove', count: 2)
+        page.all('input.collection-remove')[0].click
+        expect(page).to have_selector('input.collection-remove', count: 1)
+        expect(page).not_to have_content(work1.title.first)
+        expect(page).to have_content(work2.title.first)
+      end
+      it 'removes the second work from the list of items' do
```

Comment:
I don't believe that we need both removing the first work and removing the second work specs (feature specs are terribly expensive to run)